_posts/2020-05-29-cockpit-common-criteria.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Aligning Cockpit with Common Criteria
+author: marusak
+date: 2020-05-29
+category: announcement
+tags: cockpit common-criteria
+slug: cockpit-common-criteria
+comments: true
+---
+
+In the last few releases new features were delivered to make Cockpit correctly aligned to Common Criteria and thus making it possible to undergo the certification process in the near future.
+Having such certification would give users in security sensitive environments sufficient trust into the Web Console.
```

Comment:
It could also say something like "This certification is often required for large organizations, particularly in the public sector". Also, it may be a little presumptuous that this alone will give users "sufficient" trust -- they also need to understand the structure as a whole, like what it means to open a web port, exposing passwords  there, and such.